packages/mantine/src/blocknoteStyles.css

```
@@ -791,3 +791,9 @@ we just don't display it in CSS instead. */
 .bn-mantine .bn-badge .mantine-Chip-iconWrapper {
   display: none;
 }
+
+/* PDF export: prevent table cell overflow */
+@media print {
+  .bn-block-content table { overflow: visible !important; }
+  .bn-block-content td, .bn-block-content th { overflow: visible !important; word-wrap: break-word; }
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)